src/cli.test.ts

```diff
@@ -0,0 +1,209 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as child_process from 'child_process';
+import * as fs from 'fs';
+import * as path from 'path';
+import { promisify } from 'util';
+
+// tslint:disable: mocha-no-side-effect-code
+
+describe('axe-sarif-converter CLI', () => {
+    beforeAll(async () => {
+        await ensureDirectoryExists(testResultsDir);
+    });
+
+    it('prints help info with --help', async () => {
+        const output = await invokeCliWith('--help');
+        expect(output.stderr).toBe('');
+        expect(output.stdout).toMatchSnapshot('help_stdout');
+    });
+
+    it('prints version number from package.json with --version', async () => {
+        const output = await invokeCliWith('--version');
+        expect(output.stderr).toBe('');
+        expect(output.stdout).toBe('0.0.0-managed-by-semantic-release\n');
+    });
+
+    it('requires the -i parameter', async () => {
+        const outputFile = path.join(testResultsDir, 'overwrite_test.sarif');
+        await writeFile(outputFile, 'preexisting content');
```

Comment:
is this step relevant to the test case? I'm assuming `yargs` will check required parameters before actually try to read files, is that correct?

Reply:
Copy-paste error, fixed!